server/sonar-web/src/main/js/apps/background-tasks/components/Stats.js

```diff
@@ -38,17 +40,17 @@ export default class Stats extends React.PureComponent {
   /*:: props: Props; */
   /*:: state: State; */
 
-  handleCancelAllPending(e /*: Object */) {
-    e.preventDefault();
-    e.target.blur();
+  handleCancelAllPending = (event /*: Object */) => {
+    event.preventDefault();
+    event.currentTarget.blur();
     this.props.onCancelAllPending();
-  }
+  };
 
-  handleShowFailing(e /*: Object */) {
-    e.preventDefault();
-    e.target.blur();
+  handleShowFailing = (event /*: Object */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)